go/common/rpc/generated/enclave.proto

```diff
@@ -317,10 +317,7 @@ message AttestationReportMsg {
 }
 
 message BlockSubmissionResponseMsg {
-  ExtBatchMsg producedBatch = 1;
-  ExtRollupMsg producedRollup = 2;
   repeated SecretResponseMsg producedSecretResponses = 3;
```

Comment:
not sure what the impact is (if any) but these field index numbers should probably be updated to 1 and 2